ansible/setup.yml

```diff
@@ -6,14 +6,20 @@
   - name: find the ip of docker-machine
     local_action: shell "docker-machine" "ip" "{{docker_machine_name | default('whisk')}}"
     register: result
-    ignore_errors: true
+    when: "'environments/mac' in inventory_dir"
```

Comment:
currently we support only the 2 env `local` or `mac`. Users have to input `-i environments/local` or `-i environments/mac`. For now, `setup.yml` is strongly bind to a env, cause it needs to generate a `db_local.ini` based on template.

Another checking `docker-machine` as a command always gives a non-zero return code which also creates a "fatal" error in ansible view -> again, have to ignore this.